ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -238,33 +247,33 @@ The killed workers will be restarted automatically if the *Disable workflow* tog
 > [Caution] Problems may occur when killing workers. Only kill the workers when you are qualified to restart the stopped process actions.  
 
 
+
 ### Kill the workers of a single queue type
 
 Kill the workers of a single queue type to stop all running actions of a certain queue type.
 
 #### Prerequisites
 
-At least one queue type has been created, see [Create a queue type](#create-a-queue-type).
-
-> [Info] The *Default* queue type is predefined.
+The workflows are disabled. Disable the workflows before you kill the workers to prevent an automatic restart of the workflow processes, see [Disable the workflows](#disable-the-workflows).
```

Comment:
Ich finde, beidePrerequisites sind hier zutreffend (genommen aus Delete a queue type):
![image](https://github.com/Actindo-AG/Documentation/assets/99268392/d6b0858a-0328-4fbc-8365-1d386733d92e)

Man muss auch eine zusätzliche queue type erstellt haben, oder? Denn Default queue type kann nicht editiert werden. Wir haben eigentlich zwei Varianten: 
- At least one queue type is created...
- An additional queue type has been created...

Sollten die beide nicht vereinheitlicht werden?

Reply:
Ich kann ja nun auch auch nur den Default queue type killen. Also stimmt das erste nicht.

Reply:
Ich ändere auf "At least one...." Damit es einheitlich ist mit dem Rest der Doku